test/controllers/api/internship_offers/create_test.rb

```diff
@@ -68,6 +68,9 @@ class CreateTest < ActionDispatch::IntegrationTest
       assert_equal ['Missing sector'],
                    json_error['sector'],
                    'bad sector message'
+      assert_equal ['Missing school_track'],
+                   json_error['school_track'],
+                   'bad school_track message'
```

Comment:
```suggestion
      assert_equal ['Missing school_track'],
                   json_error['school_track'],
                   'bad school_track message'
```
doit etre setté pardéfault